doc/pki.md

```diff
@@ -0,0 +1,34 @@
+# UK Proxy Node PKI
+
+The eIDAS Proxy Node Public Key Infrastructure (PKI) is detailed in the diagram and key below.
+This PKI allows:
+
+* encryption and decryption of the eIDAS SAML request and response
+* signing and signature validation of the eIDAS SAML request and response
+* publishing of PKI metadata between the UK Proxy Node and an EU connecting country
+
+The diagram and key describe a sequential flow of of an eIDAS verify journey.
+
+![](images/proxy_node_pki.svg)
+
+Key:
+
+| Step      | Description |
+| :----:       |    :---     |
+| a   |`metadata-controller` provisions a non-extractable signing keypair in a `CloudHSM` and publishes the public key to `metadata`. It also signs the generated metadata using a keypair it generates itself.
```

Comment:
> keypair it generates itself.

This makes it sound a little bit like the metadata keypair is _litrally_ generated on the metadata controller... whereas the metadata signing keypair is also generated in the HSM.

So maybe to clarify:

> `metadata-controller` provisions a non-extractable signing keypair in a `CloudHSM` and publishes the public key to `metadata`. The published metadata is signed with a second non-extractable key whose public certificate can be shared with EU countries for the purpose of verifying metadata signatures.